app/helpers/trajectories_helper.rb

```diff
@@ -0,0 +1,2 @@
+module TrajectoriesHelper
+end
```

Comment:
if you're not use this file delete it please. :))

Reply:
right away